### ISSUE TITLE
Save feature not saving the full canvas.

### ISSUE THREAD

**Issue by jefreestyles** [zarstensen/GodotAnnotate#24]
Hi.  I'm expecting the save as image feature to have all the other elements in the scene tree.  I would then just save using Snipping Tool or another screen capture method.  Any plans to update the save feature?

![Untitled](https://github.com/zarstensen/GodotAnnotate/assets/1665452/91b258a2-9fc7-4357-af8a-d6a86b8028ff)

**Comment by zarstensen** [comment]
hi @jefreestyles, thanks for the feature request.

However, i am not sure i agree with the expectations of saving the entire scene tree as an image.
The intended usage of this feature is to convert the canvas strokes into an image, so one can apply shaders and other techniques which are not as easily done on Line2D nodes. (see #3 for more details)

You can include other nodes in the saved image by adding them as children to the canvas, however the image bounds will only account for the contents of the canvas, meaning your Icon sprite might be cut off if there are no strokes nearby on the canvas.

**Comment by jefreestyles** [comment]
I don't agree with the purpose.  The purpose of annotating is to point out things in the image to discuss or change with someone or a group of people.  If the underlying image isn't shown, then it can't be used for discussions and changes to the art / gameplay.

**Comment by zarstensen** [comment]
Ok, so what would the difference between this and a conventional screenshot via. f.ex. the shift + win + s shortcut be?

The only difference i can come up with is the editor UI being hidden in the image, and if the purpose is only for planning i am not sure it would be that important to remove these visuals?

If there is something i am missing, please say so.
